src/__tests__/to-have-style.js

```diff
@@ -36,4 +36,16 @@ describe('.toHaveStyle', () => {
     expect(() => expect(container).toHaveStyle({ fontWeight: 'bold' })).toThrowError();
     expect(() => expect(container).not.toHaveStyle({ color: 'black' })).toThrowError();
   });
+
+  test('handles null cases', () => {
+    const { getByTestId } = render(
+      <View testID="container">
+        <Text>Hello World</Text>
+      </View>,
+    );
+
+    const container = getByTestId('container');
+
+    expect(container).not.toHaveStyle({ fontWeight: 'bold' });
```

Comment:
Shouldn't the test represent a specific `null` thing? 🤔

Reply:
I should have added a more detailed description. Done now :+1: